### PR TITLE
bpo-34980: P/Invoke QueryFullProcessImageName to get process' name

### PR DESCRIPTION
**Pull request by jkloth**
https://bugs.python.org/issue34980

### CODE REVIEW

**Comment by willingc** [comment]
Thanks for the PR @jkloth. I've added the skip news label so the Windows core devs can see that there are no failing tests 👍

File: PCbuild/pyproject.props [hunk 1]
```diff
@@ -109,19 +109,33 @@
...
+public static extern bool QueryFullProcessImageName([In]IntPtr hProcess, [In]int dwFlags,
+                                                    [Out]StringBuilder lpExeName, ref int lpdwSize);
+public override bool Execute() {
+    string fullPath = Path.GetFullPath(FileName);
+    Log.LogMessage("Looking for " + fullPath, MessageImportance.Normal);
+    foreach (Process p in Process.GetProcesses()) {
+        try {
+	    int pathLength = 32768;
+	    StringBuilder pathBuilder = new StringBuilder(pathLength);
+            QueryFullProcessImageName(p.Handle, 0, pathBuilder, ref pathLength);
```
Comment (zooba) [inline PCbuild/pyproject.props]: Weird indentation going on here, and would be nice to just `continue` here if it fails.

**Review by zooba — APPROVED** [review]

File: PCbuild/pyproject.props (hunk 1 above)
Comment (jkloth) [reply PCbuild/pyproject.props]: Oops, some tabs crept in.

I'm unsure to what the `continue` you refer to.  The `return` at the end is due to the change from `type="Fragment"` to `type="Method"`.

Comment (zooba) [reply PCbuild/pyproject.props]: If `QueryFullProcessImageName` returns `false`, we don't want to bother with anything else after it.

Comment (jkloth) [reply PCbuild/pyproject.props]: Ah, yes.  Committed.

**Pull request merged**